models/apiRouteModel.js

```diff
@@ -0,0 +1,14 @@
+const fs = require("fs/promises");
+
+exports.readApiRoutes = () => {
```

Comment:
I Like your thinking here to use a readFile. You could also do this without needing a model, and just requiring in the endpoints.json in the controller.

Reply:
Would we get access to meaningful errors though? E.g. someone / something corrupts the JSON, and it doesn't include properly?